Gulpfile.js

```diff
@@ -82,7 +82,7 @@ gulp.task("build-babel-standalone", cb => {
   pump(
     [
       gulp.src(__dirname + "/packages/babel-standalone/src/index.js"),
-      webpackBuild(),
+      webpackBuild({ filename: "babel.js", library: "Babel" }),
```

Comment:
Rather than doing this, I'd prefer if this entire task was made reusable again. I'm not sure why it was changed to not be reusable. See my original implementation: https://github.com/Daniel15/babel/blob/5b9722c8255ce65584a31effe4518c0677176525/packages/babel-standalone/src/gulpTasks.js

```
function registerBabelStandaloneTask(gulp, name, exportName, path, version) {
  gulp.task("build-" + name + "-standalone", cb => {
    pump(
      [
        gulp.src(path + "/src/index.js"),
        webpackBuild(
          name + ".js",
          exportName,
          version
        ),
        gulp.dest(path),
        uglify(),
        rename({ extname: ".min.js" }),
        gulp.dest(path),
      ],
      cb
    );
  });
}
```